lib/core/test/unit/Cardano/Wallet/Primitive/CoinSelection/BalanceSpec.hs

```diff
@@ -1966,7 +1965,7 @@ isValidMakeChangeData p = (&&)
   where
     totalInputValue =
         F.fold (inputBundles p)
-            <> (F.foldMap TokenBundle.fromCoin (view #extraCoinSource p))
+            <> (TokenBundle.fromCoin (view #extraCoinSource p))
```

Comment:
```suggestion
            <> TokenBundle.fromCoin (view #extraCoinSource p)
```

Reply:
Thanks @sevanspowell. I suspect this suggestion has already been applied. (Since I can no longer find the line with the extraneous bracket.)